### DISCUSSION TITLE
docs/v1.11.x/reference/dev-env/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#2990] [Announcements]
# docs/v1.11.x/reference/dev-env/

https://kubeovn.github.io/docs/v1.11.x/reference/dev-env/

**Comment by lynn901** [comment]
在执行make release之前，最好先执行go mod tidy 和go mod vendor

**Reply by oilbeater** [reply]
make release 会调用到 build-go 里面会执行 go mod tidy. go mod vendor 属于兼容历史的功能了，暂时不会考虑

**Comment by xinsir88** [comment]
大佬好，因为我们公司内网没有互联网，所有想把咱们得帮助文档搬到我们内网，我该如何操作呢？
